### PR TITLE
Show notice for pending England applications

### PR DESCRIPTION
**Pull request by davidrapson**
Tells people with pending applications for under £10,000 in England that our priorities are different and their application might not be suitable any more.

image

image

### CODE REVIEW

**Comment by davidrapson** [comment]
Needs Welsh translation

File: controllers/apply/dashboard/index.js [hunk 1]
```diff
@@ -106,9 +108,15 @@ router.get(
                 }),
             ]);
 
+            const notice = getNotice(req.i18n.getLocale(), pendingSimple);
+            if (notice) {
+                logger.info('Notice shown on dashboard');
```
Comment (davidrapson) [inline controllers/apply/dashboard/index.js:113]: Added logs here as it's useful to see how many people see this notice.

File: controllers/apply/dashboard/lib/get-notice.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+'use strict';
+const config = require('config');
+const get = require('lodash/fp/get');
+const { oneLine } = require('common-tags');
+
+module.exports = function getNotice(locale, pendingSimple = []) {
```
Comment (mattandrews) [inline controllers/apply/dashboard/lib/get-notice.js:6]: The `pendingSimple` name here is a bit confusing, could we call it `pendingApplications` or something to make it clearer what kind of array it's operating over? (I know it's Simple apps as opposed to Standard but it's not clear it's _applications_ from the name).

Comment (davidrapson) [reply controllers/apply/dashboard/lib/get-notice.js]: Good shout, forgot to rename this.

**Comment by davidrapson** [comment]
Added welsh translation. Need to double check if this is OK to go out earlier or if it's for the 13th, in which case I'll use a different feature flag.

**Pull request merged**